README.md

```diff
@@ -31,12 +31,20 @@
 
 # Introduction
 
-The OSS Review Toolkit (ORT) aims to assist with the tasks that commonly need to be performed in the context of license
-compliance checks, especially for (but not limited to) Free and Open Source Software dependencies.
+The OSS Review Toolkit (ORT) is a FOSS policy automation and orchestration toolkit which you can use to manage your
+(open source) software dependencies in a strategic, safe and efficient manner.
 
-It does so by orchestrating a _highly customizable_ pipeline of tools that _abstract away_ the underlying services.
-These tools are implemented as libraries (for programmatic use) and exposed via a command line interface (for scripted
-use):
+You can use it to:
+
+- Generate CycloneDX, SPDX SBOMs, or custom FOSS attribution docs for your software project
+- Automate your FOSS policy using risk-based Policy as Code to do licensing, security vulnerability checks, InnerSource
+and engineering standards checks for your software project and its dependencies
+- Create a source code archive for your software project and its dependencies to comply with certain licenses or have
```

Comment:
nit: Regarding "software project and its dependencies" would it make sense to assume the audience has mutliple software projects, e.g. say "software projects and their dependencies"?

Reply:
We made it singular as:
- Checks that ORT supports are usually done on a project-by-project basis
- ORT out of the box does not have a feature to scan all software projects an person or org may have - say scanning an entire GitHub or GitLab org.